### DISCUSSION TITLE
linking a directory (windows)

### DISCUSSION THREAD

**Discussion by payrim** [TriliumNext/Trilium#1806] [Q&A] [answered]
hello
how can I open a directory from Trilium? looks like node is not accessible and i failed to write a code to open directory.

in markdown you just write "some folder" and that's it, wanted to ask is there a way to do it here? i don't want to add videos/pdfs to the data since it could get very huge and that could cause problems (also duplication on the drive and lotta other stuff) still I think attaching videos and pdf files is a great thing to do so but sometimes you just need to open a directory

anyways thanks for all the great effort!

**Comment by zadam** [comment, marked as answer]
Hi, opening a local (filesystem) directory is not currently supported.
